### PR TITLE
CMR-4472

### PR DESCRIPTION
**Pull request by ygliuvt**
Added support to create/delete service association concept in metadata-db.

### CODE REVIEW

File: metadata-db-app/int_test/cmr/metadata_db/int_test/utility.clj [hunk 1]
```diff
@@ -323,6 +330,27 @@
                            (dissoc attributes :extra-fields))]
      (concept provider-id :service uniq-num attributes))))
 
+(defn service-association-concept
+  "Creates a service association concept"
+  ([assoc-concept service uniq-num]
```
Comment (mschmele) [inline metadata-db-app/int_test/cmr/metadata_db/int_test/utility.clj]: indentation looks off here and in the arity below

File: metadata-db-app/src/cmr/metadata_db/services/messages.clj [hunk 2]
```diff
@@ -215,6 +215,11 @@
                "Variable associations are system level entities.")
           provider-id))
 
+(defn service-associations-only-system-level [provider-id]
```
Comment (mschmele) [inline metadata-db-app/src/cmr/metadata_db/services/messages.clj]: Nitpick: put argument on its own line

File: metadata-db-app/int_test/cmr/metadata_db/int_test/utility.clj (hunk 1 above)
Comment (ygliuvt) [reply metadata-db-app/int_test/cmr/metadata_db/int_test/utility.clj]: Done.

File: metadata-db-app/src/cmr/metadata_db/services/messages.clj (hunk 2 above)
Comment (ygliuvt) [reply metadata-db-app/src/cmr/metadata_db/services/messages.clj]: Looks like almost all the functions in this namespace are using this style of declaration, perhaps for the ease of readability. I am not a fan of this type of messaging functions in general and don't want my finger print to be all over this namespace. So I am only changing a couple of the functions. :)

**Pull request merged**